_templates/blog/post/post.ejs.t

```diff
@@ -0,0 +1,19 @@
+---
+to: blog-content/<%= slug %>.md
+---
+---
+title: <%= title %>
+createdAt: <%= createdAt %>
+readingTime: 10
+authorFirstName: 'Jay'
+authorLastName: 'Khatri'
+authorTitle: 'Software Engineer @ Highlight'
+authorTwitter: 'https://twitter.com/TODO'
```

Comment:
should this be templated?

Reply:
Sorry, what do you mean? Take the Twitter username as an arg when running the template?